Core/MIPS/IR/IRCompVFPU.cpp

```diff
@@ -1179,10 +1227,12 @@ namespace MIPSComp {
 				}
 			}
 			return;
-		} else if (msz == M_4x4 && !IsConsecutive4(sregs)) {
+		} else if (msz == M_4x4 && IsConsecutive4(sregs)) {
+			// Consecutive, which is harder.
```

Comment:
Yeah, we probably should. Might just as well add a Transpose4x4 instruction, the only 4x4 op in our virtual instruction set, so implementations can choose the fastest way...